Cargo.toml

```diff
@@ -113,7 +113,7 @@ lookup = { path = "lib/lookup" }
 portpicker = { path = "lib/portpicker" }
 prometheus-parser = { path = "lib/prometheus-parser", optional = true }
 tracing-limit = { path = "lib/tracing-limit" }
-value = { path = "lib/value", optional = true }
+value = { path = "lib/value", optional = false }
```

Comment:
Do you need the `optional = false`? I thought that was the default.

Reply:
It is. I'll remove this.